### ISSUE TITLE
fzf takes capitalize keybinding

### ISSUE THREAD

**Issue by mattiasb** [junegunn/fzf#1443]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [ x ] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ x ] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ x ] bash
    - [ ] zsh
    - [ ] fish

`fzf` binds `M-c` to `__fzf_cd__` [here](https://github.com/junegunn/fzf/blob/master/shell/key-bindings.bash#L117). While the functionality of `__fzf_cd__` is nice, it's also overwriting the `capitalize` command.

I like the stuff in `key-bindings.bash` and I want to use it. But would it be possible to some how configure what bindings should be set? Another works-for-me solution would be to only set keybindings to stuff that isn't taken on a standard bash installation. :)

**Comment by i-am-eva** [comment]
Not the owner but it can be hard to change these things if there is enough inertia (i.e. people used to the old binding). I agree that over-writing default bindings isn't cool but what's done is done? Adding some configuration setup would probably be a fair bit of work.

For your particular case, could you add some custom bindings after `fzf` is setup in the shell, to rebind `capitalize-word` to `M-c` and `__fzf_cd__` to whatever you wanted?

Curious what the use case for capitalising whole words on the command line is to be honest. I guess if you accidentally write `my_env = 1` and want to make it `MY_ENV = 1` faster?

**Comment by mattiasb** [comment]
Yeah inertia is definitely a thing. And indeed, redefining `M-c` after loading `fzf-keybindings.bash` is a good workaround. Thanks!

I don't remember what I'm using it for. It's just one of those standard keybinding like `M-l` and `M-u` that is so ingrained in my daily usage that I don't reflect on when and why I use them. Kind of like using `C-a` and `C-e` for `END` and `HOME` or similar. :)
